src/backend/api/reviews.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express");
+const router = express.Router();
```

Comment:
Here you do create a router! Nice, make sure you export it at the end. Since this rotuer will be used for all URLs that start with `api/reviews`, each path should start from `/` in this file.

Reply:
changes made